lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -690,23 +690,8 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
         _onAccessibilityFeaturesChanged, _onAccessibilityFeaturesChangedZone);
   }
 
-  /// Change the retained semantics data about this window.
-  ///
-  /// If [semanticsEnabled] is true, the user has requested that this function
-  /// be called whenever the semantic content of this window changes.
-  ///
-  /// In either case, this function disposes the given update, which means the
-  /// semantics update cannot be used further.
   @override
-  @Deprecated('''
-    In a multi-view world, the platform dispatcher can no longer provide apis
-    to update semantics since each view will host its own semantics tree.
-
-    Semantics updates must be passed to an individual [FlutterView]. To update
-    semantics, use PlatformDispatcher.instance.views to get a [FlutterView] and
-    call `updateSemantics`.
-  ''')
-  void updateSemantics(ui.SemanticsUpdate update) {
+  void updateSemantics(ui.SemanticsUpdate update, [ui.FlutterView? view]) {
```

Comment:
Also, why do we add the `view` parameter here? It appears to be unused? In the future (once the web engine is actually multi-view enabled) we will likely have to add it, but since `EnginePlatformDispatcher` is private API we can just do that when we need it without a breaking change?